CMakeLists.txt

```
@@ -33,11 +33,19 @@ pbuilder_prepare_project()
 set(CMAKE_CXX_STANDARD 14) # Enable c++14 standard
 
 
+########
+# HepMC3
+########
+find_package(hepmc3 REQUIRED)
+if (NOT hepmc3_FOUND)
+    message(FATAL "Unable to find HepMC3")
+endif (NOT hepmc3_FOUND)
```

```suggestion
find_package(HepMC3 REQUIRED)
if (NOT HepMC3_FOUND)
    message(FATAL "Unable to find HepMC3")
endif (NOT HepMC3_FOUND)
```

This was required in the hk-software container we use for WCSim to work. This is after doing 
```
sudo yum install epel-release
sudo yum install HepMC3 HepMC3-devel HepMC3-search HepMC3-search-devel HepMC3-interfaces-devel HepMC3-doc
```

Can you test in your setup if this works? If it doesn't, then different versions and/or installations methods of `HepMC3` provide different capitalisations of these files, which is fun...
```
    hepmc3Config.cmake
    hepmc3-config.cmake
```

I'm sure we can work around using `if` in this file, but not pleasant

FYI I installed `HepMC3-3.2.7-3` on `yum` today

---

Sure, this works for me I'll commit it to the PR. I'm surprised the lowercase version doesn't work. The `find_package` command is meant to be case-insensitive.